modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -186,22 +186,44 @@ class ApiClient(object):
             # use server/host defined in path or operation instead
             url = _host + resource_path
 
-        # perform request and return response
-        response_data = {{#asyncio}}await {{/asyncio}}{{#tornado}}yield {{/tornado}}self.request(
-            method, url, query_params=query_params, headers=header_params,
-            post_params=post_params, body=body,
-            _preload_content=_preload_content,
-            _request_timeout=_request_timeout)
-
-        self.last_response = response_data
+        try:
+            # perform request and return response
+            response_data = {{#asyncio}}await {{/asyncio}}{{#tornado}}yield {{/tornado}}self.request(
+                method, url, query_params=query_params, headers=header_params,
+                post_params=post_params, body=body,
+                _preload_content=_preload_content,
+                _request_timeout=_request_timeout)
+
+            content_type = response_data.getheader('content-type')
+
+            self.last_response = response_data
+
+            return_data = response_data
+
+            if not _preload_content:
+                {{^tornado}}
+                return return_data
+                {{/tornado}}
+                {{#tornado}}
+                raise tornado.gen.Return(return_data)
+                {{/tornado}}
+
+            if six.PY3:
+                if response_type not in ["file", "bytes"]:
```

Comment:
Can you combine these two if statements?
We don't need two levels of indentation here.
You can just use:
`if six.PY3 and response_type not in {"file", "bytes"}:`
Note: I also switched to set definition with {} for quicker lookups.